sklearn/model_selection/_split.py

```diff
@@ -1050,7 +1050,7 @@ class StratifiedShuffleSplit(BaseShuffleSplit):
     >>> y = np.array([0, 0, 1, 1])
     >>> sss = StratifiedShuffleSplit(n_iter=3, test_size=0.5, random_state=0)
     >>> sss.get_n_splits(X, y)
-    3
+    3n
```

Comment:
For some reason you have to use `ignore-files=setup.py` because trying to import sklearn/cluster/setup.py confuses the hell out of nose not sure why.